### ISSUE TITLE
Nach hinzufügen eines Nodes in einen Flow, startet RedMatic immer wieder neu

### ISSUE THREAD

**Issue by LevelOne2k** [rdmtc/RedMatic#302]
Hallo!

Ich habe gerade den Node "Fake-Roku" in einen Flow hinzugefügt, an diesen einen Change und an diesen ein RPC(?) um ein CCU Gerät hinzuzufügen. 
<img width="1015" alt="Bildschirmfoto 2020-01-07 um 16 56 13" src="https://user-images.githubusercontent.com/25386824/71908618-a6a1b300-316e-11ea-88b6-60a81c744357.png">

Das ganze hat wunderbar funktioniert und ich kann über den Fake Roku das CCU Gerät steuern.

Gerade habe ich das gleiche nochmal gemacht, nur für ein anderes CCU Gerät. Dann habe ich Deploy gedrückt und sofort wurde die Verbindung verloren und RedMatic startet seitdem immer wieder neu.

Ich habe das Log mal auf Debug gestellt, aber auch damit startet RedMatic immer wieder neu. @hobbyquaker kannst Du Dir das Logfile bei Gelegenheit bitte einmal ansehen? Meine Bude steht nämlich ohne RedMatic still ^^

Danke!

PS: Den Pioneer AVR error kannst Du unbeachtet lassen, der ist normal weil das Gerät nicht immer mit Strom versorgt ist.

**Comment by hobbyquaker** [comment]
Beende mal Node-RED im RedMatic UI. Und dann startest es im "Safe Mode", entfernst den Fakeroku Node und deployst wieder.

**Comment by LevelOne2k** [comment]
Okay das werde ich machen, danke. 

Kann man im log evtl. erkennen wieso das passiert?

**Comment by hobbyquaker** [comment]
Hast Du das Log gemailt? Ich hab nichts bekommen...

**Comment by LevelOne2k** [comment]
Ach so, hatte keine Aufforderung bekommen, deshalb noch nichts geschickt ^^ geht jetzt raus. DerAlte

**Comment by hobbyquaker** [comment]
Zum Crash führt das:
```
:58:58 raspberrymatic daemon.err node-red: 7 Jan 16:58:58 - [red] Uncaught Exception:
Jan  7 16:58:58 raspberrymatic daemon.err node-red: 7 Jan 16:58:58 - Error: listen EADDRINUSE: address already in use 192.168.191.20:8086
```

Da versucht ein Node ein Server auf Port 8086 aufzumachen (vermutlich/wahrscheinlich Fakeroku) - der Port ist allerdings schon belegt. Dass der Node diesen Fehler nicht abfängt sondern dann zum Crash führt ist schlecht, das sollte der Entwickler des Nodes abfangen, da würde ich empfehlen in dem Github Repo dieses Nodes ein Issue aufzumachen.

**Comment by naimo84** [comment]
Hallo zusammen,
ja das ist schlecht von mir umgesetzt worden... Sorry. Sobald ich Zeit finde (Arbeit, Baby und Frau 😉), werde ich den Port konfigurierbar machen und den Fehler abfangen.
Danke @hobbyquaker fürs durchforsten der log Datei

**Comment by naimo84** [comment]
Das Problem sollte gefixt bzw. Wird nun abgefangen. Sorry nochmal für solch schlechten Programmierstil...

**Comment by LevelOne2k** [comment]
@naimo84 alles gut, kann alles passieren. Vielen Dank für den fix. 

Wenn wir jetzt noch das andere welches ich Dir per Mail geschickt habe bzgl. Der Tasten mit den Lampen hinbekommen, das wäre super.

**Comment by naimo84** [comment]
Ja, das dauert immer ein wenig. Ich kann nur pushen und "request refresh" drücken. Wann node-red den refresh macht, weiß ich leider nicht
![image](https://user-images.githubusercontent.com/11535502/71979523-381c2e00-321e-11ea-95c5-897050209840.png)

**Comment by hobbyquaker** [comment]
@naimo84 Danke für die schnelle Reaktion!

**Comment by hobbyquaker** [comment]
RedMatic 9 ist als Vorabversion verfügbar (https://github.com/rdmtc/RedMatic/releases) und bewusst verschlankt: Der RedMatic-Paketmanager, die binären Pakete, die vorinstallierten Zusatz-Nodes (Dashboard, HomeKit, ZigBee, ...) und die WebApp sind entfallen. Nodes werden nur noch über den Node-RED Paletten-Manager installiert; Nodes mit nativen (binären) Abhängigkeiten lassen sich auf der CCU nicht installieren - das ist eine bekannte und akzeptierte Einschränkung (https://github.com/rdmtc/RedMatic/wiki/Node-Installation#nodes-mit-binärmodulen). Dieses Issue betrifft eine entfallene Komponente und wird daher geschlossen. Hinweise zum Umstieg: https://github.com/rdmtc/RedMatic/wiki/RedMatic-9-Migration

---
*Written by Claude Fable on behalf of hobbyquaker.*
